packages/tools/piw-utils/src/utils/PageEditorUtils.ts

```diff
@@ -71,6 +71,17 @@ export function changePropertyIn<T, TKey extends keyof T>(
     modifyProperty(modify, propertyGroups, key, nestedPropIndex, nestedPropKey);
 }
 
+export function transformGroupsIntoTabs(properties: Properties) {
+    const groups: PropertyGroup[] = [];
+    properties.forEach(property => {
+        if (property.propertyGroups) {
+            groups.push(...property.propertyGroups);
+            property.propertyGroups = [];
+        }
+    });
+    properties.push(...groups);
```

Comment:
The top level property groups are not cleared out. In the case this group has direct properties that's good. However, how will it behave in Studio when there are no direct properties?

Reply:
It doesnt render anything if there is no properties and just an empty tab (example)